plugins/services/src/js/structs/Service.js

```diff
@@ -141,6 +141,23 @@ module.exports = class Service extends Item {
     return this.get("role") || "";
   }
 
+  getQuotaRoleStats(roleName) {
+    return (this.get("tasks") || []).reduce(
```

Comment:
Have we tested this with a `Pod`? I want to make sure `tasks` is available the same way or if we need to override there like we had to do for `getRole`

Reply:
Pods are supported now- insofar as they are enforced if they have a role that matches the group